docs/tools-and-applications/airflow/airflow.md

```diff
@@ -0,0 +1,28 @@
+
+# Airflow
+
+## What is it?
+
+[**Airflow**](https://airflow.apache.org/) is a platform created by the community to programmatically author, schedule and monitor workflows.  
+It has become popular because of how easy it is to use and how extendable it is, covering a wide variety of tasks and allowing you to connect your workflows with virtually any technology.  
+Since it's a Python framework it has also gathered a lot of interest from the Data Science field.
+
+One important key concept used in Airflow is DAGs (Directed Acyclical Graphs).  
+A DAG is a graph without any cycles. In other words, a node in your graph may never point back to a node higher up in your workflow.  
+DAGs are used to model your workflows/pipelines which essentially means that you are building graphs that then get executed.
```

Comment:
Add a link to Full dag explanation: https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html

Reply:
Good call, added it